README.md

```diff
@@ -33,10 +33,10 @@ When done, the install script prints out instructions for running and starting c
 
 We also have an in-depth [setup and configuration](./doc/guide.md) guide.
 
-### Alpha Program 🐣
+### Cloud Program ☁️
 
 We're working on a cloud platform that makes deploying and managing code-server easier.
-Consider updating to the latest version and running code-server with our experimental flag `--link` if you don't want to worry about
+Consider running code-server with the beta flag `--link` if you don't want to worry about
```

Comment:
Should we complete this sentence? "if you don't want to worry about..."?

Reply:
It is already completed! Expand the part of the diff that isn't shown.